packages/datetime/src/_datepicker.scss

```diff
@@ -109,6 +109,18 @@ $header-margin: ($header-height - $pt-input-height) / 2 !default;
     display: table-row;
   }
 
+  .DayPicker-WeekNumber {
+    display: table-cell;
+    cursor: pointer;
+    width: $datepicker-day-size;
+    height: $datepicker-day-size;
+    padding-right: $pt-grid-size;
+    vertical-align: middle;
+    text-align: right;
```

Comment:
hmm this right alignment combined with the right padding is an awkward effect. @llorca maybe we just want to remove padding and center text, like we do for actual dates?

Reply:
yes, just tested in the console: turns out we can remove `width`, `height`, `padding-right`, and change `text-align: right` to `text-align: center`

Reply:
cc @franciscofabian